layer1/transaction/backend.go

```diff
@@ -129,12 +129,12 @@ func (g *group) remove(txHash common.Hash) error {
 	return nil
 }
 
-// check if a group is empty
+// isEmpty check if a group is empty
 func (g *group) isEmpty() bool {
 	return len(g.InternalGroup) == 0
 }
 
-// send a receipt inc ase this group has an unique tx or we have the receipt
+// sendReceipt sends a receipt in case this group has a unique tx, or we have the receipt
```

Comment:
Use the name fluidly in the comment sentence.

```suggestion
// sendReceipt in case this group has a unique transaction, or we have the receipt.
```